format/FormatHDF5EigerNearlyNexus.py

```diff
@@ -142,6 +142,7 @@ def create_scalar(handle, path, dtype, value):
                 shape=shape,
                 ndim=handle_orig_entry.ndim,
                 filename=handle_orig_entry.file.filename,
+                is_virtual=False,
```

Comment:
`is_virtual` better than `is_vds`?

_probably_ I would expect the latter

Reply:
This is purely for compatibility with `h5py.Dataset.is_virtual`:
https://docs.h5py.org/en/stable/high/dataset.html#h5py.Dataset.is_virtual

Reply:
That is good enough for me. I withdraw my comment.